docker-bake.hcl

```diff
@@ -179,7 +179,7 @@ target "openresty-alpine-fat" {
     dockerfile="openresty/Dockerfile-alpine"
     args = {
         OPENRESTY_VERSION = "${openresty-version}"
-        NGINX_VERSION = "${nginx-version}"
+        NGINX_VERSION = patch(openresty-version)
```

Comment:
Won't this just be `3` if openresty version is `1.25.3.1`?

Reply:
Nope, see https://github.com/coreruleset/modsecurity-crs-docker/blob/5039d9dce5689087d4d0b032476cb671f3e3913e/docker-bake.hcl#L83.

We get all the numbers up to the patch.